### PR TITLE
FFWEB-2355: convert all class memebers to static typed

### PR DESCRIPTION
**Pull request by paulfcdd**
- Solves issue: 
FFWEB-2350

- Description: 
convert all class memebers to static typed

- Tested with Magento editions/versions: 
2.4.2

- Tested with PHP versions: 
7.4.20

### CODE REVIEW

File: src/Model/Ssr/Template/Engine.php [hunk 1]
```diff
@@ -10,8 +10,7 @@
 
 class Engine implements TemplateEngineInterface
 {
-    /** @var Mustache_Engine */
-    private $engine;
+    private Mustache_Engine $engine;
```
Comment (a-laurowski) [inline src/Model/Ssr/Template/Engine.php]: can we do?
`use Mustache_Engine  as Mustache;`

and then 
`private Mustache $engine`

**Review by a-laurowski — CHANGES_REQUESTED** [review]

**Comment by a-laurowski** [comment]
I guess if you use 7.4 syntax you should remove 7.3 from the CI

File: src/Model/Ssr/Template/Engine.php (hunk 1 above)
Comment (paulfcdd) [reply src/Model/Ssr/Template/Engine.php]: done

**Comment by paulfcdd** [comment]
> I guess if you use 7.4 syntax you should remove 7.3 from the CI

There is a separate task for it - FFWEB-2361. Or you suggest to do it in this task?

**Comment by a-laurowski** [comment]
> > I guess if you use 7.4 syntax you should remove 7.3 from the CI
> 
> There is a separate task for it - FFWEB-2361. Or you suggest to do it in this task?

no, if it's in separate task, then let's ignore CI failure for now

**Review by a-laurowski — APPROVED** [review]

**Pull request merged**